interop-cats/shared/src/test/scala/zio/interop/catzSpec.scala

```diff
@@ -48,6 +53,8 @@ class catzSpec extends catzSpecZIOBase {
     "ArrowChoice[ZManaged]",
     implicit tc => ArrowChoiceTests[ZManaged[*, Int, *]].arrowChoice[Int, Int, Int, Int, Int, Int]
   )
+  // checking that this compiles(see issue #173)
+  getArrow(ZManaged.environment[Int])
```

Comment:
Could you please move these down into a syntax test object, like the other syntax test?

Reply:
moved it to  summoningInstancesTest because it is covering the Arrow instances summoning problem.